### PR TITLE
DEVDOCS-4449: [maintenance] Add Date Fields to Customs Information API

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4449]

From [PR 985](https://github.com/bigcommerce/api-specs/pull/985)

[DEVDOCS-4449]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/shipping.v3.yml [hunk 1]
```diff
@@ -313,77 +313,46 @@ components:
...
+      title: customsInformationRequest
+      description: Data about the customs information object.
       type: object
+      x-internal: false
       properties:
-        name:
-          description: Display name for shipping method.
-          example: Flat Rate
+        product_id:
+          description: The id of the product which the customs information data will apply to.
```
Comment (markcmurphy) [inline reference/shipping.v3.yml:323]:
```suggestion
          description: The product ID to which the customs information data applies.
```

File: reference/shipping.v3.yml [hunk 2]
```diff
@@ -313,77 +313,46 @@ components:
...
+          example: 77
+        country_of_origin:
           type: string
-        type:
-          title: Shipping Method Type
-          example: perorder
+          description: 'The country of manufacture, production, or growth represented in ISO 3166-1 alpha-2 format.'
+          example: US
+        commodity_description:
+          description: Description that provides information for customs to identify and verify shapes physical characteristics and packaging of each shipment.
```
Comment (markcmurphy) [inline reference/shipping.v3.yml:332]:
```suggestion
          description: Description that provides information for customs to identify and verify the shapes, physical characteristics, and packaging of each shipment.
```

File: reference/shipping.v3.yml [hunk 3]
```diff
@@ -313,77 +313,46 @@ components:
...
           type: string
+          minLength: 0
+          maxLength: 100
+          example: Baseball caps
```
Comment (markcmurphy) [inline reference/shipping.v3.yml:336]: This question might be outside our purview, but do the commodity descriptions need to adhere to a specific standard like those listed here: https://dataweb.usitc.gov/classification/commodity-description/HTS/4? Disregard if NBD.

Comment (bc-andreadao) [reply reference/shipping.v3.yml:336]: Might be for a different ticket

**Review by markcmurphy — APPROVED** [review]

**Review by markcmurphy — APPROVED** [review]

**Pull request merged**